src/main/java/cat/udl/eps/softarch/demo/config/DBInitialization.java

```diff
@@ -39,6 +39,22 @@ public void initializeDatabase() {
                 user.encodePassword();
                 userRepository.save(user);
             }
+            if (!userRepository.existsById("admin")) {
+                User admin = new User();
```

Comment:
We can keep this for the moment but later, when the Admin class is available, it should be changed to new Admin()...